src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshot.cs

```diff
@@ -118,7 +116,7 @@ private DefaultProjectSnapshot(ProjectSnapshotUpdateContext update, DefaultProje
             Version = other.Version;
         }
 
-        public override ProjectExtensibilityConfiguration Configuration { get; }
+        public override RazorConfiguration Configuration => HostProject?.Configuration;
```

Comment:
The churn here is due to the fact that we don't need to compute this in the background anymore, we always get to know up front what the version is.